src/test/java/games/strategy/util/UtilTest.java

```diff
@@ -0,0 +1,28 @@
+package games.strategy.util;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class UtilTest {
+
```

Comment:
Great to see the test case :+1: 
null and empty string might be interesting test cases to add, probably those should throw IllegalArgumentException.